src/main/java/creativei/controller/InquiryController.java

```diff
@@ -3,25 +3,28 @@
 
 import com.fasterxml.jackson.databind.ObjectMapper;
 import creativei.enums.ExceptionType;
+import creativei.exception.InvalidParamRequest;
 import creativei.manager.InquiryManager;
 import creativei.vo.InquiryVo;
 import creativei.vo.ResponseObject;
 import org.apache.commons.lang3.StringUtils;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
 import javax.servlet.http.HttpServletRequest;
 @CrossOrigin(origins = "http://localhost:8100")
+@ControllerAdvice
```

Comment:
Why are we using this annotation?

Reply:
it is cross origin resource sharing  so that if client as in our development part it is now on different domain to connect it to the server it is used